resources/web/map/js/trajectory_raw.js

```diff
@@ -77,16 +79,38 @@ function setupLayers() {
             }
         });
     }
+    if (map.getSource(`prot`) != null) return;
+    map.addSource(`prot`, {
+        type: 'geojson',
+        cluster: false,
+        data: {
+            type: 'FeatureCollection',
```

Comment:
we probably dont need a feature collection here>
since its only one point (current) on drongo